src/main/java/org/influxdb/dto/Point.java

```diff
@@ -385,4 +387,51 @@ private StringBuilder formatedTime() {
     return sb;
   }
 
+  public Point tag(final String tagName, final String value) {
+
```

Comment:
not at the moment, i just defined some checkstyle rules, which are not covering all as we can see.
But i am thinking about introducing automatic formatting with for example: https://github.com/google/google-java-format

But this can only be done at the very beginning of a sprint.